.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         board:
           - bluemicro840_v1
-          - nice_nano
```

Comment:
Can we keep the v1 in the build actions for existing users, so that they can access the settings reset uf2 and default shield keymaps for testing?

Reply:
@petejohanson What are your thoughts on this? The reset uf2s are interchangeable, but the default shield builds are definitely up for consideration.

Reply:
I would say we keep it in the list for now. At least until we can get more modular stuff in, and our refinements to the builds for PRs versus `main` for which shield/board combos we build.